samples/subsys/video/capture/src/main.c

```diff
@@ -110,8 +135,8 @@ int main(void)
 			return 0;
 		}
 
-		printk("\rGot frame %u! size: %u; timestamp %u ms",
-		       frame++, vbuf->bytesused, vbuf->timestamp);
+		printk("\rGot frame %u! size: %u; timestamp %u ms\n", frame++, vbuf->bytesused,
```

Comment:
Unrelated change?

Reply:
It disturbed me a bit to view the logs so I fixed it as well but yes it should be in another commit, will do